File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):  
    answer = ''
    
    for i in range(len(my_string)) :
        if i == num1 :            
            answer += my_string[num2:num2+1]
        elif i == num2 :
            answer += my_string[num1:num1+1]
        else :
            answer += my_string[i:i+1]
    return answer

File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.js
function solution(my_string, num1, num2) {
    var answer = '';
    const charArr = my_string.split('');
    [charArr[num1], charArr[num2]] = [charArr[num2], charArr[num1]];
    answer = charArr.join('');
    return answer;
}